package.json

```diff
@@ -104,7 +104,8 @@
     "test": "node scripts/test.js --env=jsdom",
     "precommit": "lint-staged",
     "release": "buck-trap -a",
-    "staging-deploy": "read -p 'Version? (see above) ' V && test $V && ansible-playbook -i deploy/hosts -K deploy/deploy.yml --limit=staging --extra-vars=\"sitename=staging.nxt.lizard.net version=v$V\"",
+    "staging-deploy-2": "read -p 'Version? (see above) ' V && test $V && ansible-playbook -i deploy/hosts -K deploy/deploy.yml --limit=staging2 --extra-vars=\"sitename=staging.nxt.lizard.net version=v$V\"",
+    "staging-deploy": "read -p 'Version? (see above) ' V && test $V && ansible-playbook -i deploy/hosts -K deploy/deploy.yml --limit=staging --extra-vars=\"sitename=nxt3.staging.lizard.net version=v$V\"",
```

Comment:
moet de (toekomstig) nieuwe staging server ook nog veranderd worden in
* configuration/staging-configuration.json en
* testplan.org ?